### PR TITLE
chore(deps): upgrade AJV to vesion 7

### PR DESCRIPTION
**Pull request by realityking**
This probably requires some iteration but I think it's a good base to start a discussion how upgrading to AJV 7 would look like for `jest-json-schmema`.

- #23 should be merged before this one as it removes code that'd be modified in this PR
- [Strict mode](https://github.com/ajv-validator/ajv/blob/v7.0.2/docs/strict-mode.md) is disabled. It might make sense to enable it by default (the user could always override it) but it'd require more changes from users when upgrading. Thoughts?
- Related to the above, currently the test suite causes a lot of warnings when run in strict mode. These could be fixed independent of the decision above (actually independent of upgrading to AJV 7). Should they?
-  I've included `ajv-formats` by default to approximate the experience of AJV 6. Maybe this should be left to the user?

### CODE REVIEW

**Review by 10xLaCroixDrinker — DISMISSED** [review]

File: matchers/toMatchSchema.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ const formatForPrint = (input, displayType = true) => {
 
 function buildToMatchSchema(ajv) {
   // eslint-disable-next-line no-underscore-dangle
```
Comment (infoxicator) [inline matchers/toMatchSchema.js:76]:
```suggestion
```

**Review by infoxicator — DISMISSED** [review]

**Review by nellyk — DISMISSED** [review]

**Comment by nellyk** [comment]
@realityking  thank you for your contributions #22 and #23  have been merged and released, could you please resolve the conflicts on this PR and we can review it

**Comment by realityking** [comment]
@nellyk Rebased the PR and addressed @infoxicator's comment

**Comment by nellyk** [comment]
Thank you 🙏  for your contributions

**Review by nellyk — APPROVED** [review]

**Comment by realityking** [comment]
Thanks @nellyk.

Did you have a look at the 3 questions I listed in the original post? What are your thoughts?

**Comment by realityking** [comment]
Still relevant

**Comment by realityking** [comment]
I pushed a rebase branch (same code) and removed your review in the process @nellyk 🙈 

Also FYI, AJV 8 is out. Unfortunately it requires some more changes to tests. I suggest getting this merged first and I'll be happy to contribute a PR for AJV 8. That'll make the review of the changes for each release easier.

**Comment by realityking** [comment]
Still relevant

**Comment by nellyk** [comment]
@realityking thank you for contributions i'll try to get some eyes on this. Apologies for the delays

**Review by 10xLaCroixDrinker — APPROVED** [review]

**Comment by 10xLaCroixDrinker** [comment]
We should go to version 8

**Review by JAdshead — APPROVED** [review]

**Comment by 10xLaCroixDrinker** [comment]
Bumping to 8 in #41

**Pull request merged**